sdk/dotnet/Iot/TopicRule.cs

```diff
@@ -305,8 +323,13 @@ public InputList<Inputs.TopicRuleDynamodbv2Args> Dynamodbv2s
             set => _dynamodbv2s = value;
         }
 
-        [Input("elasticsearch")]
-        public Input<Inputs.TopicRuleElasticsearchArgs>? Elasticsearch { get; set; }
+        [Input("elasticsearches")]
+        private InputList<Inputs.TopicRuleElasticsearchArgs>? _elasticsearches;
+        public InputList<Inputs.TopicRuleElasticsearchArgs> Elasticsearches
```

Comment:
This is weird to me as a former user of Elasticsearch. It's the name of the software itself so plural is odd. Something like "ElasticsearchInstances" would be more natural.

Reply:
This was fixed by changing `elasticsearches` to `elasticsearch`. This matches other fields in the same object, where it's just the name of a bunch of AWS services. I'll push the commit shortly.